api/server.js

```diff
@@ -0,0 +1,184 @@
+const express = require('express');
+
+const configureMiddleware = require('../config/middleware');
+const server = express();
+const userDb = require('../data/helpers/userDb');
+const postDb = require('../data/helpers/postDb');
+const tagDb = require('../data/helpers/tagDb');
+
+configureMiddleware(server);
+
+const upperMiddleware = (req, res, next) => {
+  req.body.name = req.body.name.toUpperCase();
+  if (req.body.name.length > 128) {
+    res.status(422).json({ message: 'The name is too long!' });
+  }
+  next();
+};
+
+server.get('/', (req, res) => {
+  res.status(200).json({ api: 'Running!' });
+});
+
+// Tags
+server.get('/api/tags/:id', async (req, res) => {
+  try {
+    const { id } = req.params;
+    const tags = await tagDb.get(id);
+    tags === undefined
```

Comment:
I like this ternary, I know they're harder to read than an `if` statement but they look cooler to me.